conda/install.py

```diff
@@ -241,7 +241,7 @@ def exp_backoff_fn(fn, *args):
             if e.errno in (errno.EPERM, errno.EACCES):
                 if n == max_retries-1:
                     raise
-                time.sleep(((2 ** n) + random.random()) * 1e-3)
+                time.sleep(((2 ** n) + random.random()) * 1e-1)
```

Comment:
I don't think we're going to be hitting the kernel that much. I'd love to see some statistics showing the average required backoff time and count for typical scenarios.

And do we really need rand at all? I suppose if it is already linked in it doesn't matter, and we already use ctypes. But I will bet a fixed backoff starting interval will suffice. Or maybe even a very simple pseudorandom source like the PID modulo 11.